easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'RPackage'
+
+name = 'MEM'
+local_commit = '6b92476'
+version = '20191023'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://github.com/cytolab/mem'
+description = "Marker Enrichment Modeling (MEM) is a tool designed to calculate enrichment scores."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/cytolab/mem/archive/']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['735cc340fb644ddc4bd760d6897d3911561ea4735db10e799ff34e36ba138a86']
+
+dependencies = [
+    ('R', '3.6.2'),
+    ('R-bundle-Bioconductor', '3.10'),
+]
+
+modextrapaths = {'R_LIBS': ''}
```

Comment:
@boegel this is done automatically for RPackage, so can be removed.

Reply:
Good point, will clean up.